Makefile

```diff
@@ -99,12 +99,12 @@ compile-thunk:
 		--outfile=${ROOTDIR}/bundles/thunk_bundle.js \
 		--inject:${ROOTDIR}/testing/injections/buffer_shim.js \
 		--format=iife \
-		--define:process.env.IS_THUNK=true \
+		--define:process.env.IN_ISOLATED_VM_OR_BROWSER=true \
 		--global-name=module.exports \
-		--target=node14;
+		--target=es2020;
 
-.PHONY: compile
-compile:
+.PHONY: compile-ts
+compile-ts:	
```

Comment:
pulling this out so that it can be used in local debugging without building the documents.

Reply:
Nice!